src/context/UserContext.js

```diff
@@ -19,6 +19,7 @@ function loginProcess(email, password) {
         alert(response.statusText)
         return null
       } else {
+        alert('Login success!!')
```

Comment:
We don't have to show this alert I believe. Instead, we redirect the user to the item list page. So, we can remove this line.